django_mongodb/base.py

```diff
@@ -139,6 +140,7 @@ def _connect(self):
         self.connection = MongoClient(
             host=settings_dict["HOST"] or None,
             port=int(settings_dict["PORT"] or 27017),
+            uuidRepresentation="standard",
```

Comment:
This is only necessary because `adapt_json_value()` isn't using `encoder`.

Reply:
Sorry, I'm not quite understanding this. The encoder is an optional parameter in JSONField and can be None. So, how should we handle UUIDs? Should we manage them manually?

Reply:
Normally trying to save them will raise an error (test_invalid_value). They work with a custom encoder because the uuid is serialized to a string (test_custom_encoder_decoder).

Reply:
Ah right, yeah, understood. Other thing that I noticed: the key has to be strings. So I have to adapt the dictionary. I can add an encoder to transform data in the same way that json.dumps does.